### PR TITLE
TW-944  Bulk export option removed from ScratchPad in the latest versions

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/getting-started/using-scratch-pad.md [hunk 1]
```diff
@@ -50,16 +50,11 @@ While you are in the Scratch Pad, you can:
 
 ## Exporting data from the Scratch Pad
 
-When you leave the Scratch Pad, your data remains there and isn't automatically transferred to your workspace. But you can export the data from the Scratch Pad as JSON, then import it to a workspace.
+When you leave the Scratch Pad, your data remains there and isn't automatically transferred to your workspace. But you can export individual collections and environments from the Scratch Pad as JSON, then import it to a workspace.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/using-scratch-pad.md:53]:
```suggestion
When you leave the Scratch Pad, your data remains there and isn't automatically transferred to your workspace. But you can export individual collections and environments from the Scratch Pad as JSON, then import them to a workspace.
```

File: src/pages/docs/getting-started/importing-and-exporting-data.md [hunk 2]
```diff
@@ -235,4 +235,4 @@ The [Export page](http://go.postman.co/me/export) shows information about your e
 
 When the export is ready, you will receive an email with link to download a zipped file with the data dump. You can also download the file from the Export page using **Download**.
 
-> Data exports from the Scratch Pad are downloaded directly instead of being emailed to you. See [Exporting data from the Scratch Pad](/docs/getting-started/using-scratch-pad/#exporting-data-from-the-scratch-pad) for more details.
+> You can only export individual collections and environments from the Scratch Pad. You can't make bulk data exports of all collections and environments at once.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/importing-and-exporting-data.md:238]:
```suggestion
> You can export individual collections and environments from the Scratch Pad. You can't make bulk data exports of all collections and environments at once.
```

File: src/pages/docs/getting-started/using-scratch-pad.md [hunk 3]
```diff
@@ -50,16 +50,11 @@ While you are in the Scratch Pad, you can:
...
-To export saved Scratch Pad data, do the following:
+For more information on exporting a single collection or an environment, see [Exporting Postman data](/docs/getting-started/importing-and-exporting-data/#exporting-postman-data).
 
-1. Go to the Scratch Pad.
-1. Select the settings icon <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-settings-v9.jpg#icon" width="16px"> in the header, then select **Settings > Data**.
-1. Select **Export Data**.
-1. Choose a directory in which to save the file, then select **Save**.
-
-You can also export a single collection or an environment. For more information, see [Exporting Postman data](/docs/getting-started/importing-and-exporting-data/#exporting-postman-data).
+> You can only export individual collections and environments from the Scratch Pad. You can't make bulk data exports of all collections and environments at once.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/using-scratch-pad.md:57]:
```suggestion
> You can export individual collections and environments from the Scratch Pad. You can't make bulk data exports of all collections and environments at once.
```

File: src/pages/docs/getting-started/importing-and-exporting-data.md (hunk 2 above)
Comment (MatthewPinsker-postman) [inline src/pages/docs/getting-started/importing-and-exporting-data.md:238]: This is minor, but what do you think about moving this note below line `215`? My thinking is that a user using Scratch Pad might land on the "Exporting data dumps" section attempting to bulk export their data. Moving the note towards the top of the section might give them their answer sooner.

**Review by mark-anthony-postman — APPROVED** [review]
Made a few minor suggestions, but otherwise LGTM!

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good! I added one minor suggestion.

**Pull request merged**